sbi/samplers/ode_solvers/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""
+Neural ODEs for sampling.
+"""
+
+import torch.nn as nn
+from torch import Tensor
+
+from sbi.samplers.ode_solvers.base import NeuralODE, NeuralODEFuncType
+from sbi.samplers.ode_solvers.zuko_ode import ZukoNeuralODE
+
+__all__ = [
+    "NeuralODE",
+    "NeuralODEFuncType",
+    "ZukoNeuralODE",
+    "build_neural_ode",
+]
+
+
+def build_neural_ode(
+    f: NeuralODEFuncType,
+    net: nn.Module,
+    mean_base: Tensor,
+    std_base: Tensor,
+    backend: str = "zuko",
+    t_min: float = 0.0,
+    t_max: float = 1.0,
+    **kwargs,
+) -> NeuralODE:
```

Comment:
we usually do not have functions in `__init__.py`. move this to `net_builders.py`?